Kickstarter-iOS/Views/PledgeScreenCTAContainerView.swift

```diff
@@ -0,0 +1,189 @@
+import KsApi
+import Library
+import PassKit
+import Prelude
+import UIKit
+
+private enum Layout {
+  enum Button {
+    static let minHeight: CGFloat = 49.0
+    static let minWidth: CGFloat = 162.0
```

Comment:
Wondering if we need a `minWidth` 🤔  On the phone this can probably just fill the stackview with spacing and margins and we could send a max width for ipad. We could set `fillEqually` on the stackview for these buttons.